web/src/Sidebar.tsx

```diff
@@ -172,6 +288,16 @@ class Sidebar extends PureComponent<SidebarProps> {
   }
 }
 
+function timeDiff(start: string, end: string): Date {
+  let t1 = moment(start)
+  let t2 = moment(end)
+  return t2.diff(t1)
+}
+
+function formatDuration(dur: Date): string {
```

Comment:
this is pretty hacky, maybe leave a note/todo. i imagine we want at minimum the ability to display durations of 1m+ as like mm:ss